package/MDAnalysis/topology/PDBParser.py

```diff
@@ -303,6 +308,21 @@ def _parseatoms(self):
         masses = guess_masses(atomtypes)
         attrs.append(Masses(masses, guessed=True))
 
+        # Guessed attributes
+        # Need to pull elements from Atom names
+        # Need to update dependent on the treatment of non-physical
+        # elements (such as the CG atoms) or missing elements as
+        # per consensus in the community
+        if all(elements):
+            elements = guess_types(elements)
+            attrs.append(Elements(elements))
+        else:
+            warnings.warn("Element information absent or inadequate.")
+            miss_elements = [ '' for i in range(len(elements))]
```

Comment:
Okay, so just give a warning and exit the loop? This would lead the `u.atoms.element ` call to fail, so I should change my test case for that accordingly then

Reply:
Yeah, at least to me, that would be the easiest option. You probably would want to document this behaviour in the class docstring though.

Reply:
Sure. Also, the code fragment for the alternative of treating the non-physical elements I wrote is available in some of the earlier commits. It might be useful to save time while tackling the issue in the near future. Should I comment the code and keep it in the file, or should I just remove it (like in this commit)? Just asking to ensure that whenever this issue pops up, it would be resolved quickly

Reply:
You might get some trouble with the tests with the expected attributes, as IIRC the base testing class actually checks to see that every `expected_attr` is available, and that attributes not in `expected_attrs` or `guessed_attrs` are not there (you'll probably want to check this). So you might have to manually modify the attribute list for the files where you are/n't expecting elements.